introduction_to_amazon_algorithms/forecasting_services_comparison/forecast_example.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<p>For markdown headers, use <code>#</code> only for the title, then <code>##</code> for subsections, <code>###</code> for sub-subsections, etc. It's important to only have one top-level header as a title for the notebook, so it's rendered correctly on the website. I.e. for this specific header, change <code># Preparation</code> to <code>## Preparation</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>